docs/embroider-css-modules-temporary/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "docs-app-for-embroider-css-modules-temporary",
+  "version": "0.1.0",
+  "private": true,
+  "description": "docs-app for embroider-css-modules-temporary",
+  "keywords": [
+    "css",
+    "css-modules",
+    "emberjs",
+    "embroider",
+    "glint",
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ijlee2/embroider-css-modules.git"
+  },
+  "license": "MIT",
+  "author": "Isaac J. Lee",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint \"app/**/*.css\" --cache",
+    "lint:css:fix": "stylelint \"app/**/*.css\" --fix",
+    "lint:dependency": "ember dependency-lint",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "lint:types": "glint",
+    "prelint:types": "type-css-modules --src app",
+    "start": "ember serve",
+    "test": "ember test --test-port=0"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.1",
+    "@ember/test-helpers": "^2.9.3",
+    "@embroider/compat": "^2.1.1",
+    "@embroider/core": "^2.1.1",
+    "@embroider/router": "^2.0.0",
+    "@embroider/webpack": "^2.1.1",
+    "@gavant/glint-template-types": "^0.3.3",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@glint/core": "^1.0.2",
+    "@glint/environment-ember-loose": "^1.0.2",
+    "@glint/environment-ember-template-imports": "^1.0.2",
+    "@glint/template": "^1.0.2",
+    "@tsconfig/ember": "^2.0.0",
+    "@types/qunit": "^2.19.5",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
+    "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.0.1",
+    "ember-a11y-testing": "^5.2.0",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "~4.12.1",
+    "ember-cli-app-version": "^6.0.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-dependency-lint": "^2.0.1",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-css-modules": "^2.0.1",
```

Comment:
Note, `ember-css-modules` is installed in `docs-app-for-embroider-css-modules-temporary`.

As a result, we know that an app may depend on `ember-css-modules` and `embroider-css-modules-temporary`.